src/apps/Permissions/PermissionsIdentityBadge.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { isAnyEntity, isBurnEntity } from '../../permissions'
+import { EthereumAddressType } from '../../prop-types'
+import LocalIdentityBadge from '../../components/IdentityBadge/LocalIdentityBadge'
+
+function PermissionsIdentityBadge({ entity, ...props }) {
+  const entityLabel = isAnyEntity(entity)
+    ? 'Any account'
+    : isBurnEntity(entity)
+    ? 'Discarded'
+    : entity
+  return <LocalIdentityBadge entity={entityLabel} {...props} />
+}
+PermissionsIdentityBadge.propTypes = {
```

Comment:
Should we include the `LocalIdentityBadge` `prop types` here?

Reply:
Done https://github.com/aragon/aragon/pull/899/commits/e8736c17a2ff3ca68c6c5834077924150c67836c